dpgen2/op/prep_caly_model_devi.py

```diff
@@ -48,7 +48,7 @@ def get_input_sign(cls):
         return OPIOSign(
             {
                 "task_name": Parameter(str),
-                "template_slice_config": Parameter(dict),
+                "expl_config": BigParameter(dict),
```

Comment:
I would prefer naming the key by "config" to keep consistent with the input config of other calypso OPs